section4/exercises/tweet.js

```diff
@@ -9,5 +9,28 @@ tweet object instances _behave_ as expected.
 */
 
 class Tweet {
-
-};
\ No newline at end of file
+  constructor(author, content, timeStamp, numberOfLikes, comments){
+    this.author = author;
+    this.content = content;
+    this.timeStamp = timeStamp;
+    this.numberOfLikes = numberOfLikes;
+    this.comments = comments;
+  }
+addComment() {
+  // is there a way to have the string be inputed at the method? ex. celebrityPost.addComment("your so funny");?
```

Comment:
Explore making this one dynamic... how can you use a parameter to make the comment that is pushed depend on what is input into the function when it is invoked?